src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -38,22 +38,30 @@ namespace NLog.Targets
     using System;
     using System.Collections.Generic;
     using System.ComponentModel;
-    using System.Configuration;
+
     using System.Data;
     using System.Data.Common;
     using System.Globalization;
     using System.Reflection;
     using System.Text;
-    using System.Transactions;
+
     using NLog.Common;
     using NLog.Config;
     using NLog.Internal;
     using NLog.Layouts;
+#if !NETSTANDARD
+    using System.Configuration;
+    using System.Transactions;
     using ConfigurationManager = System.Configuration.ConfigurationManager;
+#endif
 
     /// <summary>
     /// Writes log messages to the database using an ADO.NET provider.
     /// </summary>
+    /// <remarks>
+    /// - Transactions aren't in .NET Core: https://github.com/dotnet/corefx/issues/2949
```

Comment:
I'm not sure if that is still the case in .NET Core 2.

Reply:
You were right. It is not needed for NetStandard 2.0